app/admin/users.rb

```diff
@@ -11,6 +11,10 @@
     column :last_name
 
     actions
+
+    panel I18n.t('active_admin.charts.sign_ups_per_month') do
```

Comment:
shouldn't we add this to the dashboard instead? 
Can we use the new beautiful gem `groupdate` to substitute the concern `WithStatsScopes` that I had added to do this groupings in the dashboard?

Reply:
`WithStatsScopes` does not group, it filters, so I think its good enough, I'll dig into the gem to see if it could replace `WithStatsScopes`. Regarding where it should be, I like it better here, but maybe someone from the design team could have a more solid opinion. If you insist I'll move it.

Reply:
ahhh you're right!